nvtabular/ops.py

```diff
@@ -19,7 +19,6 @@
 import cudf
 import numpy as np
 from cudf._lib.nvtx import annotate
-
```

Comment:
I think you need this blank line here for the isort check .  Can you add the precommit hook to run isort/black/flake8 automatically? https://github.com/NVIDIA/NVTabular/pull/69

Reply:
Yes, it works with other operators. I am adding an example .py file where I use dropna in the workflow. Can you please review?